extensions/wikia/UserLogin/js/UserBaseAjaxForm.js

```diff
@@ -126,6 +128,20 @@
 	};
 
 	/**
+	 * User has signed up successfully but they haven't confirmed their email address yet.
+	 */
+	UserBaseAjaxForm.prototype.onUnconfirmedEmailResponse = function () {
+		$.get(window.wgScriptPath + '/wikia.php', {
+			controller: 'UserLoginSpecial',
+			method: 'getUnconfirmedUserRedirectUrl',
+			format: 'json',
+			username: this.inputs.username.val()
+		}, function (json) {
+			window.location = json.redirectUrl;
```

Comment:
Hm, I suppose so. I did a quick search and it looks like there's pros and cons for both with and without the `href`, but none of them are very compelling in this use case. Would it make you feel better if I changed it? :)